packages/std/src/errors/std_error.rs

```diff
@@ -526,6 +526,15 @@ impl DivideByZeroError {
     }
 }
 
+#[derive(Error, Debug, PartialEq, Eq)]
+pub enum CheckedMultiplyRatioError {
+    #[error("Denominator must not be zero")]
+    DivideByZero {},
+
+    #[error("Multiplication overflow")]
+    Overflow {},
```

Comment:
```suggestion
    DivideByZero,

    #[error("Multiplication overflow")]
    Overflow,
```

We could simplify the enum to cases without brackets since we don't plan associated data with them. Also since this is not serialized, we don't need to bother about JSON representation consistency.

Reply:
Yeah, JSON serialization habits.